src/main/resources/config.yml

```diff
@@ -209,3 +209,12 @@ reinforcement_damageMultiplier: 365
 #  connection_timeout: 10000
 #  idle_timeout: 600000
 #  max_lifetime: 7200000  
+
+# Ideally should be a power of two
+activity-map-resolution: 512
+# Radius around each area to also refresh
+activity-map-radius: 1
```

Comment:
So resolution sets the size to 512x512. I don't understand what the radius is for? Is it saying it will refresh the 512x512 areas next to the one you're in?

Reply:
Radius means it will refresh the area you are in (512x512) and all the areas around that, so a radius of one would cover a block of 3x3 around you, or 1536x1536. So the answer to your question is yes.
This can also be useful for if you change the resolution - I think 512 is a bit big, and 128x128 would be better personally. In that case you could set the radius to 6 to cover the same area (1536x1536)